glossary.md

```diff
@@ -33,8 +33,34 @@ In order to be considered Uptane-compliant, an implementation MUST follow all of
 
 *Repository:* A server containing metadata about images, and sometimes the images themselves. Other data may be stored on the repository to be accessed by ECUs during the update process.
 
+*Root of Trust:* Component that needs to always behave in the expected manner because its
+misbehavior cannot be detected.(Source: https://www.iso.org/standard/66510.html from ISO/IEC 11889-1:2015 Information technology — Trusted platform
+module library — Part 1: Architecture)
```

Comment:
```suggestion
*Root of Trust:* Component that needs to always behave in the expected manner because its misbehavior cannot be detected. (Source: https://www.iso.org/standard/66510.html from ISO/IEC 11889-1:2015 Information technology — Trusted platform module library — Part 1: Architecture)
```